extensions-contrib/prometheus-emitter/src/main/java/org/apache/druid/emitter/prometheus/Metrics.java

```diff
@@ -58,13 +61,22 @@ public DimensionsAndCollector getByName(String name, String service)
     }
   }
 
-  public Metrics(String namespace, String path)
+  public Metrics(String namespace, String path, boolean isIncludeHost, boolean isServiceAsTag)
   {
     Map<String, Metric> metrics = readConfig(path);
     for (Map.Entry<String, Metric> entry : metrics.entrySet()) {
       String name = entry.getKey();
       Metric metric = entry.getValue();
       Metric.Type type = metric.type;
+
+      if (isIncludeHost) {
```

Comment:
I am not sure if we need these flags. Can't we always include these dimensions?

I guess it would create a separate time series though according to Prometheus docs:
> Changing any label value, including adding or removing a label, will create a new time series.

Reply:
These flags are used to maintain compatibility.